include/swift/AST/DiagnosticsParse.def

```diff
@@ -248,7 +248,7 @@ ERROR(static_var_decl_global_scope,none,
       "%select{%error|static properties|class properties}0 may only be declared on a type",
       (StaticSpellingKind))
 ERROR(computed_property_no_accessors, none,
-      "computed property must have accessors specified", ())
+      "%select{computed property|subscript}0 must have accessors specified", (unsigned))
```

Comment:
Since you are going to use this with a boolean condition, using `bool` here is probably more clear.

Reply:
Thanks for the help. I fixed it.